catalog.html

```diff
@@ -0,0 +1,321 @@
+          </p>
+          <p class="hotel-card-rating">
+            <img class="rating-card-stars" src="" alt="3 звезды">
+            <span class="rating-card-text">Рейтинг: 8,9</span>
+          </p>
+          </li>
+
+          <li class="hotel-card">
+          <img class="hotel-card-image" src="" alt="Фото отеля">
+          <p class="hotel-card-info">
+            <h3 class="hotel-card-title">Villas at Poco Diablo</h3>
+            <span class="hotel-card-type">Гостиница</span>
+            <span class="hotel-card-price">От 2000 ₽</span>
+          </p>
+          <p class="hotel-card-buttons">
+            <a class="hotel-card-button button" href="#">Подробнее</a>
+            <button class="hotel-card-button favorites" type="button">В избранное</button>
+          </p>
+          <p class="hotel-card-rating">
+            <img class="rating-card-stars" src="" alt="2 звезды">
+            <span class="rating-card-text">Рейтинг: 9,2</span>
+          </p>
+          </li>
+
+          <li class="hotel-card">
+          <img class="hotel-card-image" src="" alt="Фото отеля">
+          <p class="hotel-card-info">
+            <h3 class="hotel-card-title">GreenTree Inn</h3>
+            <span class="hotel-card-type">Гостиница</span>
+            <span class="hotel-card-price">От 3000 ₽</span>
+          </p>
+          <p class="hotel-card-buttons">
+            <a class="hotel-card-button button" href="#">Подробнее</a>
+            <button class="hotel-card-button favorites" type="button">В избранное</button>
+          </p>
+          <p class="hotel-card-rating">
+            <img class="rating-card-stars" src="" alt="2 звезды">
+            <span class="rating-card-text">Рейтинг: 9,2</span>
+          </p>
+          </li>
+        </ul>
+
+        <button class="button pagination-show-more" type="button">Показать ещё</button>
+        <ul class="pagination">
+          <li class="pagination-item">
+            <a class="pagination-link pagination-current">1</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">2</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">3</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">4</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">5</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">6</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">7</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">8</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">9</a>
+          </li>
```

Comment:
А там же по макету промежуточных номеров нет. Если считать, что там кнопка, которая на JS подгрузит дополнительные варианты, то нужно добавить понятную подпись. Либо с помощью ариа, либо засунуть внутрь span, который потом будет скрыт стилями
```suggestion
          <li class="pagination-item">
            <button class="pagination-link" type="button" aria-label="Больше вариантов">…</button >
          </li>
```

Reply:
По разметке понял, не совсем ясно с логикой. Т.е. сама кнопка с подписью "..."  и ариа-лейблом, она как бы "холостая"? Или для нее на Джаве свой функционал предполагается? 
Например:
Пользователь кликает на 1,2,3,4,10 или "Посмотреть ещё" - подгружаются варианты гостиниц;
Пользователь кликает на "..." - меняется диапазон, типа "5,6,7, ..., 10";

Reply:
@skvrtz все не очевидные вещи трактуются на усмотрение разработчика, но мне кажется второй вариант. 
Потому что если бы она было "холостой", то логичнее было бы выделить каким-то цветом, что она не активная.